<launch>
    <include file="$(find camera)/launch/tag_world.launch"/>
    <include file="$(find turtlebot3_slam)/launch/turtlebot3_slam.launch">
        <arg name="slam_methods" value="gmapping"/>
    </include>
    <include file="$(find tagstore)/launch/tagstore.launch"/>
    <node pkg="localization" name="map_handler" type="map_handler.py" output="screen"/>
    <node pkg="transformations_odom" name="tf_odom" type="poseconversions.py" output="screen"/>
    <node pkg="camera" name="camera_node" type="camera.py" output="screen"/>
</launch>